### ISSUE TITLE
Support for introduction/overview text

### ISSUE THREAD

**Issue by forana** [jbauschatz/ServiceWizard#18]
Somehow allow a string to be set that counts as introductory text - this would be placed into a comment block at the start of the angular, a header section in the markdown, and a section in the HTML.

**Comment by jbauschatz** [comment]
I think this could be the purpose of "Title" and "Description" in the annotation on a class - especially if we're going to use the same annotation on service methods. Then Title and Desc will be required wherever the annotation is used, and will be used for comment generation in a consistent manner.

So

```
@ServiceWizard(
     title="Foobar Service"
     description="This service provides crud function for Foobars"
)
class FoobarService {
     @ServiceWizard(
          title="Get Foobars"
          description="Gets every single Foobar that ever was"
     )
     @GET
     public List<Foobar> getFoobars() {

    }
}
```

?

**Comment by forana** [comment]
Talking about an overview for the whole set of classes, not just one.

**Comment by jbauschatz** [comment]
Ok. For drop wizard that could be an annotation on the main class that
actually assembles all the resources. Perhaps there shod be an "API" class
that contains the Service representation as well as intro text

On Saturday, September 5, 2015, forana notifications@github.com wrote:

> Talking about an overview for the whole set of classes, not just one.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jbauschatz/ServiceWizard/issues/18#issuecomment-137986536
> .

## 

Cheers,
John Charles

**Comment by forana** [comment]
I don't think we should require any class just for generation purposes. Extending that, I don't think this project's role is to dictate structure at all - the only interaction this should have with the application's code is to mark docs on the resource classes themselves.

Not sure if this is an MVP feature, but it's worth thinking about. Possibly related to #21.
